packages/store/addon/-private/system/references/has-many.ts

```diff
@@ -76,6 +76,7 @@ export default class HasManyReference extends Reference {
   destroy() {
     if (CUSTOM_MODEL_CLASS) {
       unsubscribe(this.#token);
+      this.#token = null as unknown as object;
```

Comment:
This is a bit out of band, but tell me more about this double `as`.

Reply:
if something already has a type, you can't recast it without first casting to unknown